README.md

```diff
@@ -27,7 +27,9 @@ skipper make test
 - `PORT` - Service listen port
 - `HTTPS_KEY_FILE` - tls key file path
 - `HTTPS_CERT_FILE` - tls cert file path
-- `ASSISTED_SERVICE_URL` - URL to use to query assisted service for image information
+- `ASSISTED_SERVICE_SCHEME` - protocol to use to query assisted service for image information
+- `ASSISTED_SERVICE_HOST` - host to use to query assisted service for image information
+- `REQUEST_AUTH_TYPE` - determines how the auth token should be passed to the assisted service - either `header` or `param`
```

Comment:
Maybe add a word or two about the distinction between them? "header" being HTTP Authorization header bearer token, "param" being a URL query parameter